### DISCUSSION TITLE
node attributes on ha_cluster system-role.

### DISCUSSION THREAD

**Discussion by esanchezvela** [linux-system-roles/linux-system-roles.github.io#100] [Q&A] [answered]
I'm trying to setup an HA cluster using the ha_cluster role, so far, I have defined a couple of resources, stonith device and cluster properties following the documentation provided with the role, but just can't find a way to define node attributes and I'm wondering whether there is a direct way to get it done. 

I've looked for a special variable or maybe a configuration option in the inventory but so far, I haven't found a way. I'd appreciate assistance on how to accomplish it.

**Comment by richm** [comment]
@tomjelinek any ideas?
@esvmicrosoft is this specific to Azure?

**Comment by tomjelinek** [comment]
Node attributes are not supported by the ha_cluster role yet.

**Comment by richm** [comment, marked as answer]
@esvmicrosoft please file an issue directly with the ha_cluster role at https://github.com/linux-system-roles/ha_cluster/issues/new

**Comment by esanchezvela** [comment]
done, thnx.
